src/duplo_resource/jit.py

```diff
@@ -24,28 +25,35 @@ def aws(self, nocache: bool = None):
 
     # check if admin or choose tenant
     if self.duplo.isadmin:
-      path = "adminproxy/GetJITAwsConsoleAccessUrl"
+        path = "adminproxy/GetJITAwsConsoleAccessUrl"
     else:
-      t = self.duplo.load("tenant")
-      tenant = t.find(self.duplo.tenant)
-      path = f"subscriptions/{tenant['TenantId']}/GetAwsConsoleTokenUrl"
+        t = self.duplo.load("tenant")
+        tenant = t.find(self.duplo.tenant)
+        path = f"subscriptions/{tenant['TenantId']}/GetAwsConsoleTokenUrl"
     
     # try and get those creds
     try:
-      if nc:
-        sts = self.duplo.get(path).json()
-      else:
-        sts = self.duplo.get_cached_item(k)
-        if self.duplo.expired(sts.get("Expiration", None)):
-          raise DuploExpiredCache(k)
+        if nc:
+            sts = self.duplo.get(path).json()
+        else:
+            sts = self.duplo.get_cached_item(k)
+            if self.duplo.expired(sts.get("Expiration", None)):
+                raise DuploExpiredCache(k)
```

Comment:
Looks like you have your local formatter set to four spaces a a tab. Can you please set that to two spaces per tab so that the spacing is consistent with the rest of the project.

Reply:
Fixed